bot/cogs/fun.py

```diff
@@ -39,6 +39,14 @@ def __init__(self, bot: Bot):
         if "dabato" in message.content:
             await message.add_reaction("🤔")
 
+        # React FBI OPEN UP if message contains trigger words.
+        triggers_list = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
+        react_list = ["🇫", "🇧", "🇮", "⚫", "🇴", "🅿️", "🇪", "🇳", "⚪", "🇺", "🇵"]
+        for _ in range(0, len(triggers_list) - 1):
```

Comment:
I'd suggest not using `_`, since it's conventionally used as a throwaway variable. Use something like `trigger`.

Reply:
I will do